### PR TITLE
Fixed issue where asyncValidateWithText would not trigger animation

### PR DESCRIPTION
**Pull request by naqi**
I tried to trigger a custom view animation from AsyncValidateWithText but since the Validation was being performed in an Async block it would not trigger or do it in its own time.

Hence, I moved 
BOOL isValid = _asyncTextValidationBlock(text)
inside 
dispatch_async(dispatch_get_main_queue()

I am calling my own network Async task from it so the networking call is still in background and not effecting UI.

### CODE REVIEW

File: BZGFormField/BZGFormField.m [hunk 1]
```diff
@@ -107,18 +111,18 @@ - (void)setup
...
                                       cancelButtonTitle:@"Ok"
                                       otherButtonTitles:nil];
     self.alertView.delegate = self;
-
+    
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(textFieldTextDidChange:)
                                                  name:UITextFieldTextDidChangeNotification object:nil];
-
+
```
Comment (0thernet) [inline BZGFormField/BZGFormField.m:125]: adding lots of whitespace :(

File: BZGFormField/BZGFormField.m [hunk 2]
```diff
@@ -153,9 +157,10 @@ - (void)asyncValidateWithText:(NSString *)text
     
     __weak typeof(self) weakSelf = self;
     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
-        BOOL isValid = _asyncTextValidationBlock(text);
+
         dispatch_async(dispatch_get_main_queue(), ^{
-        [weakSelf.leftIndicator.layer removeAnimationForKey:kValidationAnimationKey];
+                    BOOL isValid = _asyncTextValidationBlock(text);
```
Comment (0thernet) [inline BZGFormField/BZGFormField.m:162]: fix indentation

**Comment by 0thernet** [comment]
can you remove the added whitespace and fix the indentation?

**Comment by naqi** [comment]
Yea, sorry, Xcode happened.... Ill send another pull request and remove all unwanted whitespaces.

**Pull request closed without merging**